src/SFA.DAS.AssessorService.Application/Handlers/Staff/GetLearnerDetailHandler.cs

```diff
@@ -99,15 +101,24 @@ public async Task<LearnerDetailResult> Handle(GetLearnerDetailRequest request, C
                 IsPrivatelyFunded = certificate?.IsPrivatelyFunded,
                 CertificateId = certificate?.Id,
                 PrintStatusAt = certificate?.CertificateBatchLog?.StatusAt,
-                ReasonForChange = certificate?.CertificateBatchLog?.ReasonForChange
+                ReasonForChange = certificate?.CertificateBatchLog?.ReasonForChange,
+                ContactName = certificateData.ContactName,
+                ContactOrganisation = certificateData.ContactOrganisation,
+                ContactAddLine1 = certificateData.ContactAddLine1,
+                ContactAddLine2 = certificateData.ContactAddLine2,
+                ContactAddLine3 = certificateData.ContactAddLine3,
+                ContactAddLine4 = certificateData.ContactAddLine4,
+                ContactPostCode = certificateData.ContactPostCode,
+                LastUpdatedAt = new[] { certificate?.ToBePrinted, certificate?.CreatedAt, certificate.LatestChange(), 
```

Comment:
The certificate.LatestChange is exactly what you are after here - just return that

Reply:
done.